<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GY-521 User Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="gyroscrope_graph" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1182</width>
      <height>277</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="accelerometer_graph" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>292</y>
      <width>1182</width>
      <height>277</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>10</y>
      <width>391</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
